OpenRA.Mods.Common/Activities/Air/TakeOff.cs

```diff
@@ -37,7 +37,7 @@ public override Activity Tick(Actor self)
 			var destination = rp != null ? rp.Location :
 				(hasHost ? self.World.Map.CellContaining(host.CenterPosition) : self.Location);
 
-			if (NextActivity == null)
+			if (nextActivity == null)
```

Comment:
Again I think mixing `nextActivity` and `NextActivity` is dangerous.  Can we please keep this as `NextActivity == Parent`?